firebase-firestore/src/main/java/com/google/firebase/firestore/AggregateQuerySnapshot.java

```diff
@@ -87,8 +84,6 @@ public Object get(@Nonnull AggregateField aggregateField) {
    * @param countAggregateField The count aggregation for which the value is requested.
    * @return The result of the given count aggregation.
    */
-  // TODO(sumavg): Remove the `hide` and scope annotations.
-  /** @hide */
   @RestrictTo(RestrictTo.Scope.LIBRARY)
```

Comment:
I think this RestrictTo should be removed. This member should be in the public API.